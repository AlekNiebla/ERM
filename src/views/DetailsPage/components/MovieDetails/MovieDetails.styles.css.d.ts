export const main: string;
export const poster: string;
export const details: string;
export const title: string;
export const category: string;
export const yearAndDuration: string;
export const year: string;
export const duration: string;
export const description: string;
