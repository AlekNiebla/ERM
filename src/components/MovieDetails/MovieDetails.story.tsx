import * as React from "react";
import { storiesOf } from "@storybook/react";
import MovieDetails from "./MovieDetails";

storiesOf("Components / MovieDetails", module).add("default", () => <MovieDetails />);
