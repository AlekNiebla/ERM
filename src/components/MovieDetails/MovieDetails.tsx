import * as React from "react";

const MovieDetails = () => <span>MovieDetails</span>;

export default MovieDetails;
