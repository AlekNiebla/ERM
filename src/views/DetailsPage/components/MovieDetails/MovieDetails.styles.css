.main {
}
.poster {
}
.details {
}
.title {
}
.category {
}
.yearAndDuration {
}
.year {
}
.duration {
}
.description {
}
