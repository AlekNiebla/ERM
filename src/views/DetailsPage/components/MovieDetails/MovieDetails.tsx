import * as React from "react";

const MovieDetails = () => (
  <div>
    <img src="" alt="Movie Title" />
    <div>
      <span>Movie Title</span>
      <span>Oscar-winning Movies</span>
      <div>
        <span>1994</span>
        <span>154 min</span>
            </div>
            <span>Waving together three stories featuring a burger-loving hit man, his philosophical partner and a washed-up boxer, Quenting Tarantino influenced a generation of filmmakers with this crime caper's stylized, ove-the-top violence and dark komic spirit.</span>
    </div>
  </div>
);

export default MovieDetails;
