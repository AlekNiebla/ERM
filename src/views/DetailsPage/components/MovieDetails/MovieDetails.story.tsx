import * as React from "react";
import { storiesOf } from "@storybook/react";
import MovieDetails from "./MovieDetails";

const containerStyle = {
  backgroundColor: "#1e0f75",
  display: "flex",
  justifyContent: "center",
  alignItems: "center",
  padding: "200px"
};

storiesOf("Components / MovieDetails", module).add("default", () => (
  <div style={containerStyle}>
    <MovieDetails />
  </div>
));
