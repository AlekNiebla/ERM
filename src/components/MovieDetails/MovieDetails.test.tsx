import * as React from "react";
import { shallow } from "enzyme";
// components
import MovieDetails from "./MovieDetails";

describe("MovieDetails component...", () => {
  it("...renders without errors", () => {
    const wrapper = shallow(<MovieDetails />);

    expect(wrapper.find(MovieDetails)).toBeTruthy();
    // snapshot
    expect(wrapper).toMatchSnapshot();
  });
});
