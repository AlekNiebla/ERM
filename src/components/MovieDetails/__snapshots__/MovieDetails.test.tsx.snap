// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails component... ... matches snapshot 1`] = `ShallowWrapper {}`;

exports[`MovieDetails component... ...renders without errors 1`] = `ShallowWrapper {}`;
